css/00-settings.css

```diff
@@ -6,6 +6,21 @@
 	--color-brandGrey: #968c83;
 	--color-brandYellow: #e1cd00;
 	--color-brandBlue: #005eb8;
+
+	/* Clip Path Varaibles */
+	--topLeftX: 0;
+	--topLeftY: 0;
+	--topRightX: 0;
+	--topRightY: 0;
+	--bottomRightX: 0;
+	--bottomRightY: 0;
+	--bottomLeftX: 0;
+	--bottomLeftY: 0;
+	--rotate: 0;
+	--topPadding: 1rem;
```

Comment:
Oooh boy this is exciting. 

A suggestion - can this be:

	--topPadding: 2rem;
	--bottomPadding: 2rem;
	--leftPadding:2rem;
	--rightPadding: 2rem;

It's feels less cramped.

Reply:
Also, can we slide these over a bit. In my quick look I couldn't figure out how to code a demo of what I mean, but I'd like them to drift to the left a bit. Partly so the edge of the letters in the headers and paragraphs seem to line up, or if the header goes further to the left works to interrupt the flow and catch the eye which, even if breaking a text layout rule, is in the spirit of what headers do. (Am I over explaining?)

<img width="805" alt="Screen Shot 2019-08-11 at 8 09 53 PM" src="https://user-images.githubusercontent.com/67271/62841287-53d1d700-bc74-11e9-9592-327ab87fb5b0.png">

Reply:
For example, I did get this working and I love how it turned out:

<img width="1188" alt="Screen Shot 2019-08-11 at 8 18 49 PM" src="https://user-images.githubusercontent.com/67271/62841373-55e86580-bc75-11e9-94b9-320166477f5a.png">

I just added

	margin-left: -4em;

To line 16 of c4aa-clippaths.css

Reply:
@slambert is this ready to merge then?

Reply:
Nope nope! Above are just suggestions/examples.